### DISCUSSION TITLE
AQLM: Extreme Compression of Large Language Models via Additive Quantization

### DISCUSSION THREAD

**Discussion by joseph777111** [ggml-org/llama.cpp#5984] [Ideas]
Official Pytorch repository for Extreme Compression of Large Language Models via Additive Quantization 

https://arxiv.org/pdf/2401.06118.pdf 

https://github.com/Vahe1994/AQLM

Thoughts?

**Comment by vladfaust** [comment]
New boss in town: AQLM + [PV tuning](https://arxiv.org/abs/2405.14852).

See https://huggingface.co/ISTA-DASLab/Mistral-7B-v0.1-AQLM-PV-2Bit-1x16-hf (2.51GB for 7B model). Immensely useful for edge.
